Header header
foveated_stereo_ros/PointClouds point_clouds
foveated_stereo_ros/Covariance[] covariances
foveated_stereo_ros/Information[] informations

